tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -246,7 +247,7 @@
  /* Legacy wrapper */
  void
  PyOS_AfterFork(void)
-@@ -5040,6 +5058,8 @@
+@@ -5037,6 +5055,8 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes